webclient/src/views/view/view-view.scss

```diff
@@ -196,24 +194,55 @@
         }
     }
 
+    .menu {
+        padding: 0;
+        box-shadow: none;
+
+
+        .menu-item button {
+            text-align: left!important;
+            border: 0 transparent!important;
+            width: 100%;
+        }
+
+        .menu-item a,
+        .menu-item label,
+        .menu-item button {
+            cursor: pointer;
+            user-select: none;
+        }
+    }
+
     #rooms-overview {
+
+
```

Comment:
```suggestion
        #rooms-overview-select .menu-item {
            padding: 0;
            
            & .icon-arrow-right {
                margin-right: 4px;
            }
        }
```

This looks slightly better in my opinion. What do you think?

Before:
![image](https://user-images.githubusercontent.com/7429408/157310382-571bebc6-934a-4889-ae2c-756efeb92b7f.png)

Now:
![image](https://user-images.githubusercontent.com/7429408/157310410-3bb9458e-e489-4828-b5ee-a67cf61c83eb.png)